File: src/PHP/Creational Patterns/Factory/CreditCards/Creators/TitaniumCreditCardCreator.php
<?php

namespace CreditCards\Creators;

use CreditCards\Products\Titanium;

class TitaniumCreditCardCreator implements ICreditCardCreator
{
    public function createCreditCard($model, $limit, $annualCharge): Titanium
    {
        return new Titanium($model, $limit, $annualCharge);
    }
}

File: src/C#/Creational Patterns/Factory/CreditCards/Creators/TitaniumCreditCardCreator.cs
using CreditCards.Products;
using System;
using System.Collections.Generic;
using System.Text;

namespace CreditCards.Creators
{
    public class TitaniumCreditCardCreator : ICreditCardCreator
    {
        public CreditCard CreateCreditCard(string model, decimal limit, decimal annualCharge)
           => new Titanium(model, limit, annualCharge);
    }
}
